doc/user/zebra.rst

```diff
@@ -559,6 +559,153 @@ presence of the entry.
+   ---------- -------------------- ------------------------ ------------
+    End        USP                  2001:db8:1:1:1::/80      static
+    End.DT4    table 1              2001:db8:1:1:3::/80      bgp
+    End.DT4    table 2              2001:db8:1:1:7::/80      bgp
+
+.. index:: segment-routing
+.. clicmd:: segment-routing
+.. index:: srv6
+.. clicmd:: srv6
+.. index:: locators
+.. clicmd:: locators
+
+   User can enter the SRv6 configuration node with ``segment-routing`` and
+   ``srv6`` commands in configure mode. If there is some SRv6-locator exist,
+   SRv6 feature is looked enabled and this affects running-config.
+
+   User can enter the Locators node with ``locators`` command.
+   in srv6 configure mode.
+   After entering locators node, user can configure one or multi SRv6-locators.
+
+.. index:: locator NAME
+.. clicmd:: locator NAME
+.. index:: prefix X:X::X:X/M [function-bits-length 32]
+.. clicmd:: prefix X:X::X:X/M [function-bits-length 32]
+
+   Following example console log shows the typical configuration of
+   SRv6 data-plane. After a new SRv6 locator, named LOC1, is created,
+   LOC1's prefix is configured as ``2001:db8:a:a::/64``.
+
+   If user or some routing daemon allocates new SID on this locator,
+   new SID will allocated in range of this prefix.
+
+   For example, if some routing daemon creates new SID on locator
+   (``2001:db8:a:a::/64``), Then new SID will be
+   ``2001:db8:a:a:7::/80``, ``2001:db8:a:a:8::/80``, and so on.
+
+   Each locator has default SID that is SRv6 local function "End".
+   Usually default SID is allocated as ``PREFIX:1::``.
+   (``PREFIX`` is locator's prefix)
+   For example, if user configure the locator's prefix as
+   `2001:db8:a:a::/64`, then default SID will be `2001:db8:a:a:1::`)
+
+   The function bits range is 16bits by default.
+   If operator want to change function bits range, they can configure
+   with ``function-bits-length`` option.
+
+::
+
+   router# configure terminal
+   router(config)# segment-routinig
+   router(config-sr)# srv6
+   router(config-srv6)# locators
+   router(config-srv6-locs)# locator LOC1
+   router(config-srv6-loc)# prefix 2001:db8:a:a::/64
+
+   router(config-srv6-loc)# show run
+   ...
+   segment-routing
+    srv6
+     locators
+      locator LOC1
+       prefix 2001:db8:a:a::/64
+      !
+   ...
+
+.. index:: function [auto | X:X::X:X] action ACTION ACTIONARG DEVICE
+.. clicmd:: function [auto | X:X::X:X] action ACTION ACTIONARG DEVICE
+
+  FRR can configure explicit SRv6 function setting.
+  User can configure ``ip -6 route add encap seg6local`` on FRR's cli.
+  SID can be configure both dynamic allocation and explicit allocation.
+  If you chose explicit allocation, specified SID value must be in range
+  of locator's prefix.
+
+Following console log shows you both dynamic allocated SID and
+Explicit allocated sid.
+
+::
+
+  router(config-srv6-loc)# prefix 2001:db8:a:a::/64
+  router(config-srv6-loc)# function auto action endx 2001::10 eth0
```

Comment:
where does the keyword "endx" come from - is there a table of the words that correspond to the action enum?

Reply:
Ah.... 

As a result of the patch fix, this CLI has not been added yet. (It was the content of the next PR.)

I just delete these lines. And one of the next PRs will have these docs.

Reply:
I fixed it on this force-push.
https://github.com/FRRouting/frr/compare/79546d9705221b02a181489afaec51f3a8674de2..45f5426e90e20791496a47257ce9aaac2a96b879